tests/qos/test_ecn_config.py

```diff
@@ -0,0 +1,227 @@
+"""
+Tests for verifying the configuration on the DUT
+to match the expected WRED probability configuration
+"""
+
+import logging
+import time
+import pytest
+from tests.common.cisco_data import is_cisco_device
+import json
+
+
+pytestmark = [
+    pytest.mark.disable_loganalyzer,
+    pytest.mark.topology('any')
+]
+
+
+def enable_serviceability_cli(duthost, show_cmd):
```

Comment:
If the original state is disabled, will you restore it to the original state at the end of the test?

Reply:
No. It is not disabled again.  But I don't see the need for disabling it as most of the platform cli won't work without it.

Reply:
@kevinskwang please confirm.

Reply:
suggest to recover to the original state, unless you can upload the results to show the subsequent cases will not be impacted after enabling the diag shell in this new case.

Reply:
Hello @kevinskwang 

We internally also keep the dshell enabled for other testcases and the addition was to enable it in case its not enabled by default.

I have captured the below logs.

Step 1: Ran the current PR test test_ecn_config
and then
Step 2: Ran the testQosSaiPfcXoffLimit

I am attaching the results for the same. 

[tr_2024-04-18-18-12-25_pfcxoff.txt](https://github.com/sonic-net/sonic-mgmt/files/15080211/tr_2024-04-18-18-12-25_pfcxoff.txt)
[tr_2024-04-18-17-05-11 ecn_config.txt](https://github.com/sonic-net/sonic-mgmt/files/15080216/tr_2024-04-18-17-05-11.ecn_config.txt)